src/main/java/johnny/EmptyDescriptionException.java

```diff
@@ -0,0 +1,14 @@
+package johnny;
+
+public class EmptyDescriptionException extends Exception{
```

Comment:
```suggestion
public class EmptyDescriptionException extends Exception {
```
Missing whitespace between Exception and the curly brace.